.gitignore

```
@@ -5,6 +5,9 @@ __pycache__/
 
 database.db
 ./database.db
+*.db
+./*.db
```

```suggestion
*.db
```

요거 그냥 하나만 해줘도 됩니다 ㅋㅋ